Content.Server/Anomaly/Effects/InnerBodyAnomalySystem.cs

```diff
@@ -187,8 +186,10 @@ private void OnMobStateChanged(Entity<InnerBodyAnomalyComponent> ent, ref MobSta
         if (args.NewMobState != MobState.Dead)
             return;
 
-        if (EntityManager.HasComponent<PendingZombieComponent>(args.Target))
-            return; // Keep your anomaly when becoming a zombie
+        var ev = new BeforeRemoveAnomalyOnDeathEvent();
+        RaiseLocalEvent(args.Target, ev);
```

Comment:
```suggestion
        RaiseLocalEvent(args.Target, ref ev);
```

Reply:
Okay! These changes are done in https://github.com/space-wizards/space-station-14/pull/33867/commits/9c29960f2fcb77b49c112e2b4807ef99103c8149